codespell_lib/data/dictionary.txt

```diff
@@ -14261,7 +14266,7 @@ rennovation->renovation
 rentime->runtime
 rentors->renters
 reoccurrence->recurrence
-reocurring->recurring
+reocurring->recurring, reoccurring,
```

Comment:
Nice catch. I think `reoccurring` should be the first result

Reply:
OK. Done.